binderhub/tests/test_repoproviders.py

```diff
@@ -37,45 +37,81 @@ def test_spec_processing(spec, raw_user, raw_repo, raw_ref):
     assert raw_ref == unresolved_ref
 
 
-async def test_zenodo():
-    spec = '10.5281/zenodo.3242074'
-
+@pytest.mark.parametrize('spec,resolved_spec,resolved_ref,resolved_ref_url,build_slug', [
+    ['10.5281/zenodo.3242074',
+     '10.5281/zenodo.3242074',
+     '3242074',
+     'https://doi.org/10.5281/zenodo.3242074',
+     'zenodo-3242074'],
+    # 10.5281/zenodo.705645 -> This DOI represents all versions, and will always resolve to the latest one
+    # for now it is 3242074
+    ['10.5281/zenodo.3242073',
```

Comment:
```suggestion
    ['10.5281/zenodo.705645',
```
I think, based on your comment above.

Reply:
but https://doi.org/10.5281/zenodo.705645 and https://doi.org/10.5281/zenodo.3242073 are different repos. https://doi.org/10.5281/zenodo.3242073  is the url for all versions of the repo which is created by Binder Project and used in tests

Reply:
sorry, my comment is wrong :(

Reply:
i corrected the comment